test/unit/stereocamera/test_phobosstereocamera.py

```diff
@@ -16,6 +16,105 @@
 import cv2
 from phase.pyphase.stereocamera import CameraDeviceInfo, createStereoCamera
 from phase.pyphase.stereocamera import CameraDeviceType, CameraInterfaceType
+from phase.pyphase import readImage
+
+
+def test_PhobosStereoCamera_setflip():
```

Comment:
Should be one function for each line in test plan so would combine this test with the parameters test